### ISSUE TITLE
Issue with live streams

### ISSUE THREAD

**Issue by evgenibers** [sealedtx/java-youtube-downloader#12]
If video is live stream or already finished stream - video.videoWithAudioFormats() and video.videoFormats() are empty

**Comment by sealedtx** [comment]
@evgenibers I dont know if it is possible to download live video, could you provide some links so I can test it?

**Comment by evgenibers** [comment]
Sure, here some example: https://www.youtube.com/watch?v=5h4df9Vy_pM
It seems that for those videos page source code dosen't have json array "formats"

**Comment by sealedtx** [comment]
@evgenibers I just tested your [video ](https://www.youtube.com/watch?v=5h4df9Vy_pM) (on f6b4b26ed9597c752462c551fc2cbf649c2f31ab) and I succefully got formats. Are you sure about this link? There is a bug which blocks downloading live stream even if they are already finished - throwing `YoutubeException.LiveVideoException("Can not download live stream")` . But I just made a commit with check if stream is finished and probably fix it. I'm ready to release it but waiting for more links from you to test bug with empty formats. Thank you
